src/CMakeLists.txt

```diff
@@ -11,10 +11,12 @@ add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/VolumeControl2.h ${CMAKE_C
 
 set(SRC
 	main.cpp
+	SysInfo.cpp
```

Comment:
To keep file names consistent with the C++ convention, this should be lowercase: sysinfo.cpp and sysinfo.h (yeah, that's different from the QML convention... what can I say, I didn't come up with these conventions, I probably hate them as much as you do sigh)

Reply:
(and in case that wasn't clear, keep the class named SysInfo, that follows the convention, but the file names are lowercase anyway)

Reply:
I find it odd and inconsistent, but OK.